src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -2467,3 +2467,25 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 "HealthCertificate_Info_register_section03" = "Ihr Impfnachweis wird in regelmäßigen Abständen automatisch überprüft.";
 
 "HealthCertificate_Info_disclaimer" = "Ausführliche Informationen zur Datenverarbeitung und Ihrem Einverständnis.";
+
+/* Health Certificate Details */
+
+"HealthCertificate_Details_vaccinationCount" = "Impfung %i von %i";
+
+"HealthCertificate_Details_certificate" = "Impfzertifikat";
+
+"HealthCertificate_Details_dateOfBirth" = "geboren %@";
```

Comment:
is there missing "am"? 
So that the value would be "geboren am %@"?

Reply:
No - this is what was given. I guess it will change later